### PR TITLE
Makes flashbangs no longer effective against blobs

### PR DESCRIPTION
**Pull request by Bawhoppen**
I have seen time and time again through countless rounds dickwads using flashbangs against blobs for a very minor benefit, while a large amount of unflashprotected crew try to actually fight the blob and then they all get stunned and wrecked by the blob. Its better to just remove any blob damage from flashbangs so fighting blobs is more of an actual team effort.

:cl: 
tweak: Flashbangs no longer affect blobs. Don't throw them.

/:cl:

### CODE REVIEW

**Comment by Chocobro** [comment]
might want to add a changelog for this
if the players don't know it was changed, they will continue to flashbang

**Comment by Shadowlight213** [comment]
Flashbangs USED to do a lot more to the blob. I think some of @ChangelingRain's changes nerfed this.

**Comment by Bawhoppen** [comment]
I will make a changelog I was just getting the PR up

**Comment by Robustin** [comment]
@Shadowlight213 

They still do, this is just a Bawhoppen balance meme PR. Yesterday I killed two nodes with just a couple lasers and a box of flashbangs. 

We can play the "ITS A PLAYER PROBLEM NOT A CODE PROBLEM" meme and say that its up to the crew to prevent friendly fire with flashbangs, or we can also take the common sense position that we shouldn't reduce the tools available to the crew for fighting the blob because a couple fucktards misuse them. What's next? ASSISTANT KILLED HIMSELF STANDING IN FRONT OF EMITTER: EMITTERS NO LONGER HURT BLOB PR.

No matter what position you can, this is a terrible PR.

**Comment by ChangelingRain** [comment]
I actually buffed flashbangs, I never actually nerfed them

**Comment by Bawhoppen** [comment]
oh look robustin making stupid comments again
nothing new here

**Comment by kevinz000** [comment]
Code is not how we solve problems with PLAYER SKILL.

**Comment by Bawhoppen** [comment]
it just encourages people to not cooperate

**Comment by YakumoChen** [comment]
>"Flashbanging everyone get back"
>Flashbang blob
>Let crew rush in

It's called teamwork Baw.

**Comment by PKPenguin321** [comment]
flashbangs actually straight up killing the blob never made sense and all it ever did was justify griefing the crew as security during blob rounds

baw is really petty sometimes but he hit the mark on this one

:+1:

**Comment by Shadowlight213** [comment]
>People announcing that they are flashbanging the blob instead of just throwing them, perfectly safe with bowmans and sunglasses.

I have never seen this happen tbh.

**Comment by AdamElTablawy** [comment]
what about making flashbangs more effective versus blobs to justify their use rather than removing it outright

**Comment by PKPenguin321** [comment]
Because a screen-wide insta-blob-death isn't balanced at all, especially when multiple boxes of them are given to sec roundstart?

**Comment by idontwannasignup** [comment]
I'm for it.

**Comment by Incomptinence** [comment]
Flashproof or fireproof team is pretty much my dream for what people should be hardcore cracking down on the blob with. Not with stupidly broken and perfect RND items that will probably get nerfed to hell because they screw non blobs too.

**Comment by Robustin** [comment]
@PKP They only kill regular blobs in 2 flashbangs; nodes, factories, etc. would take more than an entire box. Plus blobs are at a 76% win rate currently and you're seriously arguing that Crew needs one of their only "Not Xrays" options needs to be nerfed?

**Comment by Bawhoppen** [comment]
ill just give the armory a box of grenades to compensate

**Comment by Lumipharon** [comment]
Literally what is this pr?

Dumb people friendly fire, lets nerf the item rather than punish the retard (ic or ooc).
By the same logic bombs should be removed because sometimes people kill innocents - but guess what, they just get banned for being retarted.

Same shit with welder dudes blowing everyone up when trying to refill - don't code shit out because people are dumb.

**Comment by Robustin** [comment]
Next Week:

:cl: Bawhoppen
- Welders were dumb they barely hurt the blob and people would just get rekt by the blob and become blob zombies and kill me
- Welders also cause weldingtanks to explode which would get a large group of people killed
- Therefore, welders no longer hurt the blob

**Comment by PKPenguin321** [comment]
Nice straw man

**Comment by Lumipharon** [comment]
That's literally exactly what he's saying about flashbangs though?

**Comment by PKPenguin321** [comment]
circumstances are different though

flashbangs are generally used by the box which locks down a crew for like upwards of a minute while the blob only might lose a few tiles while welders require a fuel tank to cause a bang and even when they do cause a bang it doesn't lock down crew for as long, flashbangs are basically limited to sec while welders are the primary method of attack most of the time, etc

baw's argument and robustin's "argument" are very different, but the strawman is designed to make the arguments look to the same. because the circumstances are vastly different, however, the arguments are not actually the same. this is the definition of a strawman.

why do i still have to teach people this

**Comment by Lumipharon** [comment]
But it's exactly the same. This pr suggestes nerfing flashbangs vs blob for the pure reason that 'some players are dumb and fuck over their own team mates'.

People regularly blow up fuel tanks, because fuel tanks are REQUIRED for sustained welder use, and they set nerds on fire (in a pretty large radius now), and stun/deafen and sometimes kill.

Also flashbangs only require 2 to kill all normal blobs on screen.
Basically, stop trying to nerf shit due to player retardation.

**Comment by Bawhoppen** [comment]
@Lumipharon reread PKP's post

**Comment by Lumipharon** [comment]
I have. What have I missed?

The statement that flashbangs stun people for upwards of a minute? I pointed out that there is no logical reason to do that as it's over kill on normal blobs, and not strong enough to waste on strong blobs.

The arguments are not vastly different - they are both tools used to kill the blob, an if used by a retard, they can fuck with the crew horribly. It is as simple and straight forward as that.

**Comment by Robustin** [comment]
Almost any tactic is available against blobs, and almost any tactic can backfire and wipe out the crew instead. That's why we we have admins to regulate stupid decisions, not coders gutting gameplay to try and quarantine stupidity (hint: you can't).

**Comment by KorPhaeron** [comment]
Hugbox

**Comment by KorPhaeron** [comment]
Kill or ban dumb people instead

**Comment by Bawhoppen** [comment]
I'd love to but my admin request got rejected
